src/app/main.cpp

```diff
@@ -891,6 +891,16 @@ int main( int argc, char *argv[] )
   // Initialize the application and the translation stuff
   /////////////////////////////////////////////////////////////////////
 
+
+#if defined(Q_OS_WIN)
```

Comment:
Just curious, is `Q_OS_WIN` referring to the host at compile time or the build target?